alchemy-web/docs/getting-started.md

```diff
@@ -54,66 +133,140 @@ const app = await alchemy("my-first-app", {
 > [!NOTE]
 > Learn more about Alchemy scopes in [Concepts: Scope](./concepts/scope.md)
 
-### Step 2: Instantiate a Resource
-
-A Resource is just an async function that takes a unique id (e.g. `config`) and some properties.
+### Step 2: Create a Cloudflare Worker
 
 ```typescript
-import { File } from "alchemy/fs";
+import { Worker } from "alchemy/cloudflare";
 
-// Create a file resource
-const hello = await File("hello", {
-  path: "./hello.txt",
-  content: "hello world"
+// Create a simple worker
+const worker = await Worker("hello-worker", {
+  entrypoint: "./src/worker.ts",
+  url: true, // Enable workers.dev subdomain
 });
 
-console.log(`Created file at: ${hello.path}`);
+console.log(`Worker deployed at: ${worker.url}`);
 ```
 
 > [!NOTE]
 > Learn more about Alchemy resources in [Concepts: Resource](./concepts/resource.md)
 
 ### Step 3: Finalize the Application
 
-At the end of our script, call `finalize`.
-
 ```typescript
 // Finalize the app to apply changes
 await app.finalize();
 ```
 
-This is necessary for deleting what are called "orphaned resources" (more on that below).
+This is necessary for deleting what are called "orphaned resources" when resources are removed.
 
 > [!NOTE]
 > Learn more about finalization and destroying resources in [Concepts: Destroy](./concepts/destroy.md)
 
-## Run the Script
+### Step 4: Create the Worker Script
+
+Create a `src/worker.ts` file with your worker code:
 
-Now we simply run the script. Alchemy is just pure TypeScript, so you can run it with any JS engine, e.g. `bun`:
+```typescript
+export default {
+  async fetch(request: Request): Promise<Response> {
+    return Response.json({ 
+      message: "Hello from Alchemy!",
+      timestamp: new Date().toISOString()
+    });
+  },
+};
+```
+
+## Deploy Your Worker
+
+Run the Alchemy script to deploy your worker:
 
-```bash
+::: code-group
+
+```sh [bun]
 bun ./alchemy.run.ts
 ```
 
-You will see output similar to:
+```sh [npm]
+npx tsx ./alchemy.run.ts
+```
 
+```sh [pnpm]
+pnpm tsx ./alchemy.run.ts
 ```
-Create:  "my-first-app/dev/hello"
-Created: "my-first-app/dev/hello"
+
+```sh [yarn]
+yarn tsx ./alchemy.run.ts
 ```
 
-This indicates that Alchemy has:
-1. Identified that the resource needs to be created
-2. Successfully created the resource
+:::
+
+You will see output similar to:
 
-We now have a `./hello.txt` file in our project:
 ```
-hello world
+Create:  "my-first-app/dev/hello-worker"
+Created: "my-first-app/dev/hello-worker"
+Worker deployed at: https://hello-worker.your-subdomain.workers.dev
 ```
 
+Visit the URL to see your worker in action!
+
 > [!TIP]
 > If you're familiar with other IaC tools, this should feel similar to `terraform apply`, `pulumi up`, `cdk deploy` or `sst deploy`
 
+## Add Type-Safe Bindings
```

Comment:
I'd prefer to the be Add Bindings

And then after binding another section dedicated to the types and how to configure that am